components/style/values/specified/length.rs

```diff
@@ -1226,3 +1227,97 @@ impl LengthOrNumber {
         }
     }
 }
+
+/// A value suitable for a `min-width` or `min-height` property.
+/// Unlike `max-width` or `max-height` properties, a MinLength can be
+/// `auto`, and cannot be `none`.
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[allow(missing_docs)]
+pub enum MinLength {
```

Comment:
The other relevant differences that occur to me:

- The default for `min-width` is `0`, and the default for `max-width` is `none`. But this could be done in the property definitions. (And actually, I had this wrong; I just corrected it.)
- For `min-width`, `fit-content` is "Defined as min(max-content, max(min-content, fill-available)", whereas for `max-width`, `fit-content` is "The same as max-content" (according to MDN). But for what I add in this PR, I don't think that's really necessary.

I do think it would be less code, and I could make the change if you prefer.